File: cpp/3rd_Week/MariMencoba3/Arsitek.cpp
#include "Manusia.cpp"

class Arsitek: public Manusia{
    private:
        string kodeArsitek, kantor, tahunBekerja;

//    constructor
    public:
        Arsitek(){}

    //    set dan get method
            //set method
        void setKodeArsitek(string kodeArsitek) {
            this->kodeArsitek = kodeArsitek;
        }

        void setKantor(string kantor) {
            this->kantor = kantor;
        }

        void setTahunBekerja(string tahunBekerja) {
            this->tahunBekerja = tahunBekerja;
        }

            //get method
        string getKodeArsitek() {
            return kodeArsitek;
        }

        string getKantor() {
            return kantor;
        }

        string getTahunBekerja() {
            return tahunBekerja;
        }

        ~Arsitek(){};
};

File: java/javadpbo/src/com/sem4dpbo/week3/marimencoba3/Arsitek.java
package com.sem4dpbo.week3.marimencoba3;

//inidividual class
public class Arsitek {
    private String kodeArsitek, namaArsitek, kantor, tahunBekerja;

//    constructor
    public Arsitek(){}

//    set dan get method
        //set method
    public void setKodeArsitek(String kodeArsitek) {
        this.kodeArsitek = kodeArsitek;
    }

    public void setNamaArsitek(String namaArsitek) {
        this.namaArsitek = namaArsitek;
    }

    public void setKantor(String kantor) {
        this.kantor = kantor;
    }

    public void setTahunBekerja(String tahunBekerja) {
        this.tahunBekerja = tahunBekerja;
    }

        //get method
    public String getKodeArsitek() {
        return kodeArsitek;
    }

    public String getNamaArsitek() {
        return namaArsitek;
    }

    public String getKantor() {
        return kantor;
    }

    public String getTahunBekerja() {
        return tahunBekerja;
    }
}
